web.tf

```diff
@@ -46,7 +46,7 @@ resource "aws_security_group" "default-acc-sg" {
   }
 }
 resource "aws_instance" "acme_web" {
-  # The connection block tells our provisioner how to
```

Comment:
**Accurics remediation**
 
**[HIGH]: http port open to internet**
 
**Vulnerability**
http port open to internet
 
**Remediation**
Change cidr_block to non 0.0.0.0/0